src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,9 @@
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns="http://jboss.org/schema/arquillian"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+  <extension qualifier="openshift">
+    <property name="namespace.use.current">true</property>
```

Comment:
So basically, we cannot use an ephemeral project. We force it to be the current one.

Reply:
We can. I added this so that the test behavior doesn't change, but if I remove this line and enable image stream detection, the test runs in `itest-xxxx` namespace just fine.

I thought I could force `-Dnamespace.use.current=false` on the cmdline, but it seems Arquillian has the priorities backwards :-(

Reply:
I now updated to Cube 1.9.1 and using an ephemeral project is now easily possible.

`mvn clean verify -Popenshift,openshift-it` runs the tests in current project.

`mvn clean verify -Popenshift,openshift-it -Dnamespace.use.current=false -DenableImageStreamDetection=true` runs the tests in an ephemeral project.